src/main/java/seedu/address/logic/commands/taskcommands/DeleteTaskCommand.java

```diff
@@ -41,6 +41,7 @@ public CommandResult execute() {
 
 	        try {
 	            model.deleteTask(taskToDelete);
+	            model.clearTasksFilter();
```

Comment:
Should deleting clear the filter? If i just did a `find xyz` which returns multiple tasks, then `delete 1`, and i want to clear another `xyz` related task, I'd have to `find xyz` again.